modules/data-streams/src/test/java/org/elasticsearch/datastreams/DataStreamsStatsTests.java

```diff
@@ -269,7 +268,10 @@ private DataStreamsStatsAction.Response getDataStreamsStats(boolean includeHidde
         DataStreamsStatsAction.Request request = new DataStreamsStatsAction.Request();
         if (includeHidden) {
             request.indicesOptions(
-                new IndicesOptions(EnumSet.of(IndicesOptions.Option.ALLOW_NO_INDICES), EnumSet.of(IndicesOptions.WildcardStates.HIDDEN))
+                new IndicesOptions(
+                    request.indicesOptions().options(),
+                    new IndicesOptions.WildcardOptions.Builder(request.indicesOptions().wildcardOptions()).removeHidden(false).build()
+                )
```

Comment:
Small change in semantics, instead of creating new indices options, we only change the remove hidden flag and keep the defaults for the other parameters.

I think this is an (small) improvement.